package.json

```diff
@@ -4,10 +4,9 @@
   "description": "As part of HackReactor's software engineering curicula: Front End Capstone",
   "main": "index.js",
   "scripts": {
```

Comment:
Can we add an npm start script that just starts your backend using: 
node server/index.js?